modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerms.scala

```diff
@@ -24,7 +25,7 @@ abstract class SwaggerLogAdapter[F[_]] {
   def error(message: String): F[Unit]
 }
 
-abstract class SwaggerTerms[L <: LA, F[_]] {
+abstract class SwaggerTerms[L <: LA, F[_]] extends FunctionK[SwaggerTerm[L, ?], F] {
```

Comment:
Do you plan to de-`FunctionK` this (and `FrameworkTerm` below) as well?

Reply:
Yep. Going to do one release with the hybrid TF/Free stuff, then kill FunctionK after that. Note, this PR _is still using `Free[F, ?]`,_ as the edges are still asking for Free, but now the thing that controls whether we're using Free or TF is exactly one function.